### ISSUE TITLE
Pagination shows extra page number when near start or end of range

### ISSUE THREAD

**Issue by wyatt-troia** [Semantic-Org/Semantic-UI-React#3547]
### Steps

```
class MyPagination extends React.Component<PaginationProps> {
  static defaultProps = {
    boundaryRange: 0,
    defaultActivePage: 1,
    showEllipsis: false,
    siblingRange: 2,
  };

  render() {
    const {
      boundaryRange,
      defaultActivePage,
      showEllipsis,
      siblingRange,
      totalPages,
    } = this.props;

    return (
      <Pagination
        boundaryRange={boundaryRange}
        defaultActivePage={defaultActivePage}
        ellipsisItem={showEllipsis ? undefined : null}
        siblingRange={siblingRange}
        totalPages={totalPages}
      />
    );
  }
}

export default MyPagination;
```
I'm instantiating like this:

`<MyPagination totalPages={50} />`

And my props are coming out as expected: [React dev tools screeenshot](https://i.stack.imgur.com/MXLs2.png)

### Expected Result
I expect the number of page numbers visible to remain constant despite where in the page range the user currently is.

[Desired behavior GIF](https://imgur.com/PiWfO7g)

### Actual Result

My pagination component is showing an extra, undesired item when the user is at or near either the beginning or the end of the item list.

[Current behavior GIF](https://imgur.com/a/t0mTdGo)

### Version
0.83.0

### Testcase
[Code Sandbox](https://codesandbox.io/s/5ok1jy8w4)

**Comment by layershifter** [comment]
This issue was fixed in `0.84.0` by #3271, please confirm.

**Comment by wyatt-troia** [comment]
Confirmed. Thanks.
